lib/src/remote_frame_buffer_widget.dart

```diff
@@ -26,6 +26,9 @@ class RemoteFrameBufferWidget extends StatefulWidget {
   final Option<void Function(Object error)> _onError;
   final Option<String> _password;
   final int _port;
+  final Option<ValueNotifier<Map<String, dynamic>>> _keyEventNotifier;
+  final Option<bool> _monitorClipBoard;
```

Comment:
suggestion: can't this be a simple `bool`? Makes the using code a lot shorter and more readable as well.